.github/workflows/test.yaml

```diff
@@ -0,0 +1,69 @@
+name: Integration Tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Weekdays at 13:00 UTC = 05:00 PST / 06:00 PDT.
+    - cron: "5 4 * * 1-5"
```

Comment:
> I decided to run this on a nightly `schedule` and on `workflow_dispatch`. It takes around 10 minutes so it _could_ run on `pull_request` if we want too.

Ehhh, maybe on `pull_request` makes more sense... that will get us coverage for when the python code changes.

Reply:
Let's start scheduled/on-demand. Gotta start somewhere.